src/classes/COFF_Affiliation_TEST.cls

```diff
@@ -735,14 +735,16 @@ public with sharing class COFF_Affiliation_TEST {
     @isTest
     public static void testUpdateConnectionsOnFacultyChangeUpdateExisting() {
         setup();
+
         TDTM_ProcessControl.setRecursionFlag(TDTM_ProcessControl.registeredTrigger.COFF_Affiliation_TDTM, true);
 
         //Create Course Offering with Faculty member
         Course_Offering__c offering = new Course_Offering__c(Section_ID__c = 'BIO-101 Spring 16', Course__c = course.ID,
         Faculty__c = faculty.Id, Term__c = term.Id, Capacity__c = 200);
         insert offering;
 
-        Course_Enrollment__c conn = new Course_Enrollment__c(Contact__c = faculty.Id, Course_Offering__c = offering.Id);
+        //Create Primary Course Connection for the Faculty member
```

Comment:
I had to fix a few unit tests in this class because of the efficiency improvements. If the record didn't actually need to be updated, the unit tests were failing as they were not included in the objectsToInsert or objectsToUpdate lists.